src/components/view/Contact/ContactNotesCard/index.js

```diff
@@ -0,0 +1,31 @@
+
+import React, { Component } from 'react';
+
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+class ContactProfileCard extends React.Component {
+  constructor(props) {
```

Comment:
you can ditch the constructor, it's called with props by default if it's not specified